oauth2_provider/views/oidc.py

```diff
@@ -212,45 +212,28 @@ def _validate_claims(request, claims):
 def validate_logout_request(request, id_token_hint, client_id, post_logout_redirect_uri):
     """
     Validate an OIDC RP-Initiated Logout Request.
-    `(prompt_logout, (post_logout_redirect_uri, application), token_user)` is returned.
+    `(application, token_user)` is returned.
```

Comment:
Are you changing the return signature of a public method? That can't happen because someone else may have already overridden it in their code. #1244 where this method was first introduced was published with [Release 2.3.0](https://github.com/jazzband/django-oauth-toolkit/releases/tag/2.3.0) so it's been out there for a few months. While there's a low likelihood that someone has extended it, it would introduce a breaking change. Can you re-implement this in a new method?

You'll want to create a new method AFAIK.